specification/azsadmin/resource-manager/containerregistry/Microsoft.ContainerRegistry.Admin/preview/2019-11-01/capacity.json

```diff
@@ -0,0 +1,180 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-11-01",
+    "title": "ContainerRegistryManagementClient",
+    "description": "Admin Container Registry Management Client."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerRegistry.Admin/locations/{location}/capacity/{capacityName}": {
+      "get": {
+        "x-ms-examples": {
+          "Returns container registry capacity properties.": {
+            "$ref": "./examples/Configuration/GetCapacity.json"
+          }
+        },
+        "tags": [
+          "ContainerRegistryCapacity"
+        ],
+        "description": "Returns container registry capacity property.",
+        "operationId": "ContainerRegistryCapacity_Get",
+        "parameters": [
+          {
+            "$ref": "containerregistry.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/CapacityNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Container registry capacity property is being returned.",
+            "schema": {
+              "$ref": "#/definitions/ContainerRegistryCapacity"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "containerregistry.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerRegistry.Admin/locations/{location}/capacity": {
+      "get": {
+        "x-ms-examples": {
+          "Returns container registry capacity properties.": {
+            "$ref": "./examples/Configuration/ListCapacity.json"
+          }
+        },
+        "tags": [
+          "ContainerRegistryCapacities"
+        ],
+        "description": "Returns a list of container registry capacity properties.",
+        "operationId": "ContainerRegistryCapacity_List",
```

Comment:
same

Reply:
It is ok, we don't want to collide with Storage Api for example which also has similar operations.

Reply:
not addressed

Reply:
i made the change, but for some reason it generates the ps cmdlet with very generis name like Get_AzsCapacity.ps1 what is not appropriate for our needs, we need more specific name to not confuse operator, like: Get-AzsContainerRegistryCapacity.ps1
I will  kept this as is for now.